Formula/spotifyd.rb

```diff
@@ -20,26 +20,25 @@ class Spotifyd < Formula
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
   depends_on "dbus"
-
-  on_linux do
-    depends_on "alsa-lib"
-  end
```

Comment:
Why remove this?

Reply:
It's a dependency of the Rodio backend, which I presume was chosen because it's multi-platform. However, Rodio doesn't support M1s, so this PR switches to Portaudio, which also supports both Linux and macOS.